cdap-security/src/test/java/io/cdap/cdap/security/auth/AESCipherTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.security.auth;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class AESCipherTest {
+  protected static final Logger LOG = LoggerFactory.getLogger(TestTokenManager.class);
+
+  @Test
+  public void testEncryptionAndDecryption() throws CipherException {
```

Comment:
nit: It might be good to add some sanity tests which check failure cases, e.g. invalid padding, invalid key, etc

Reply:
Done. 

Seems not possible to trigger exception during encryption as config settings are hardcoded.

But added a test for exception in decryption by intentionally corrupting the cipher data.